src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -189,6 +189,18 @@ private void validateUpdate(final String name, final EventType eventType) throws
 
         validateName(name, eventType);
         validateSchemaChange(eventType, existingEventType);
+        eventType.setDefaultStatistic(
+                validateStatisticsUpdate(existingEventType.getDefaultStatistic(), eventType.getDefaultStatistic()));
+    }
+
+    private EventTypeStatistics validateStatisticsUpdate(final EventTypeStatistics existing, final EventTypeStatistics newStatistics) throws InvalidEventTypeException {
```

Comment:
This method is used on updating event type, and if no default_statistics provided, tries to leave previous one. If default_statistics is provided - checks that it's the same as it was on creation.